### PR TITLE
Issues 25 to 29

### PR DESCRIPTION
**Pull request by tommorris168**

### CODE REVIEW

**Review by RyanJField — APPROVED** [review]
This looks good, the release date will need to be changed when pushed to shiny apps. The only suggestion would be host the Cochrane Handbook on GitHub so that it can be updated without pushing a new version of the app. There is the option of making a repo and serving the handbook as a GitHub page so it is accessible from the web.

**Comment by tommorris168** [comment]
@RyanJField , do you mean the handbook itself or my instructions?

**Comment by RyanJField** [comment]
Sorry, your handbook.

File: README.md
Comment (RyanJField) [inline README.md:1]: Line 11 should not say beta, perhaps early release.

**Review by RyanJField — CHANGES_REQUESTED** [review]

**Review by RyanJField — APPROVED** [review]
Looks good.

**Pull request merged**